### DISCUSSION TITLE
Web console for easier managment

### DISCUSSION THREAD

**Discussion by jacksonm36** [pelican-dev/panel#221] [Ideas]
add feature like on cockpit web terminal for easier managment? as for security we  could have fail2ban integration and login again before having to acces the console.

**Comment by TekExplorer** [comment]
I don't really see this being useful. At that point, just have cockpit installed wherever you want it, and it will be way more secure.

**Comment by notAreYouScared** [comment]
We have no interest in making a web terminal for the host node machine, that is not what this panel is designed for.
